problem 1 i viggoswindchill

TheNextGen76 opened the pull request:
det här fungerar inte och jag vet inte varför linus hjälp

TheNextGen76 committed:
problem 1 i viggoswindchill

det här fungerar inte och jag vet inte varför linus hjälp

Reckysfriend commented on Lab2/Program.cs:
```diff
@@ -48,11 +47,50 @@ static void Main(string[] args)
+                                {
+                                    Console.Clear();
+                                    Console.WriteLine("\tWrite Speed\n");
+                                    Int32.TryParse(Console.ReadLine, out int userInput);
```
Console.Readline()

Reckysfriend replied:
Console.Readline()

Reckysfriend commented on Lab2/Program.cs:
```diff
@@ -48,11 +47,50 @@ static void Main(string[] args)
+                            bool subMenu = true;
+                            while (subMenu)
+                            {
+                                Console.Write("\tChoose a speed km/h or m/s:");
```
Skulle göra en Switch istället för en if else 

[1] km/h 
[2] m/s

Reckysfriend commented on Lab2/Program.cs:
```diff
@@ -48,11 +47,50 @@ static void Main(string[] args)
+                                    Console.WriteLine("\tWrite Speed\n");
+                                    Int32.TryParse(Console.ReadLine, out int userInput);
+                                    int convertMeters = (userInput * 3600) / 1000;
+                                    windSpeedsStr = " - " + convertMeters.ToString();
```
ConvertedMeters*  tänk på att vara tydlig med vad som är i, nu låter det som något som converterar den inte att det är det converterade värdet.

Reckysfriend commented on Lab2/Program.cs:
```diff
@@ -48,11 +47,50 @@ static void Main(string[] args)
+                                    else
+                                    {
+                                        Console.Clear();
+                                        Console.WriteLine("\tThis is not a WindSpeed! Please enter valid Wind Speed.\n");
```
Dubble kolla hur du skriver. Du skriver WindSpeed och Wind Speed i samma mening, välj 1 av dom.

TheNextGen76 committed:
Update Program.cs

TheNextGen76 committed:
Update Program.cs

Reckysfriend reviewed (commented):
What? ConverMeters -> ConvertedMeters

TheNextGen76 committed:
nuärdenklarmenful

den är klar nu linus men den ser lite ful ut när man skriver så om du kan formatera den är själva programmet klart.

Reckysfriend committed:
Visual Update

Added visual and did a few restructures to make sure variables were more effectivly used.

Reckysfriend committed:
Small opsi

Reading is hard

Reckysfriend committed:
Reading is very hard
